src/inet/node/inet/NodeBase.ned

```diff
@@ -120,6 +122,10 @@ module NodeBase
             parameters:
                 @display("p=149,406,row,10;q=txQueue;i=block/ifcard");
         }
+        tun: <default("TunInterface")> like ITunNic if hasTun {
```

Comment:
tun interface must be a vector, and need one tunApp for each tun interface.
See it here: https://github.com/inet-framework/inet/commit/d1d72062e6c0a6cf15b61ea29569aff8e2f6c896